src/EnableNewWebImplementation.ts

```diff
@@ -3,15 +3,28 @@ import { Platform } from 'react-native';
 let useNewWebImplementation = true;
 let getWasCalled = false;
 
+/**
+ * @deprecated the legacy implementation is no longer supported. This function will be removed in Gesture Handler 3
+ */
 export function enableExperimentalWebImplementation(
   _shouldEnable = true
 ): void {
   // NO-OP since the new implementation is now the default
+  console.warn(
+    'The legacy web implementation is deprecated and will be removed in Gesture Handler 3'
+  );
```

Comment:
Are you sure it is correct message? 😅

Reply:
Changed to `Legacy web implementation` in [fe7f358](https://github.com/software-mansion/react-native-gesture-handler/pull/3259/commits/fe7f358511ffe5a14e6c11bc9e9897733dbbecb5)

Reply:
Double-fixed in [f1d5b6d](https://github.com/software-mansion/react-native-gesture-handler/pull/3259/commits/f1d5b6d9f19a1b9bd9e603aefb846e9f8273ed5b)